docs/pykiso_2_0/pykiso_exposed.rst

```diff
@@ -0,0 +1,26 @@
+.. _pykiso_exposed:
+
+Pykiso With Exposed Interface
+-----------------------------
+
+Introduction
+~~~~~~~~~~~~
+
+On the journey of transforming pykiso to a full fledged test participant, its interface needs to be exposed and accessible by third party test runner.
+This is a experimental feature (as of today) that allow to access the `auxiliaries` via an exposed interface.
+
+The first provided interface would be REST based. The concept enable contributors to create and add their own "exposed" interface to pykiso.
```

Comment:
If using rest it will be limited to simple datatypes or ?
We have auxiliaries where we pass and retrieve python objects. 

I guess for some simple stuff it would work.